### DISCUSSION TITLE
Feature Request: Document array_agg()

### DISCUSSION THREAD

**Discussion by iangow** [duckdb/duckdb#7685] [Ideas]
Why do you want this feature?

There is an `array_agg()` function in DuckDB (I use it [here](https://iangow.github.io/sql_book/cohorts.html#reading-the-data)), but there is no documentation for it. It's not listed [here](https://duckdb.org/docs/sql/aggregates) and nothing shows up in a search for it.

**Comment by Mause** [comment]
It's documented here:  https://duckdb.org/docs/sql/functions/nested.html#:~:text=array_aggregate(list%2C%20name)

Not sure why it doesn't show up in the search though

**Reply by iangow** [reply]
That seems to be actually a different function with a confusingly similar name. The `array_agg()` (aggregate) function should appear [here](https://duckdb.org/docs/sql/aggregates).

**Comment by Mytherin** [comment]
`array_agg` is an alias for `list`, which is documented on the aggregate page. We should probably add a set of aliases there as well.

**Reply by Mytherin** [reply]
Actually the alias is already there. Perhaps this is only an issue with the search then

**Reply by iangow** [reply]
Might also help to list it in the left-hand column there, but make it clear that it just points to `list()` (coming from PostgreSQL, I just scanned that column).
